l1-contracts/src.ts/deploy.ts

```diff
@@ -49,7 +49,7 @@ export class Deployer {
     this.ownerAddress = config.ownerAddress != null ? config.ownerAddress : this.deployWallet.address;
   }
 
-  public async initialProxyDiamondCut() {
+  public async initialProxyDiamondCut(validiumMode: boolean) {
```

Comment:
I think it would be more explicit if you'll provide `PubdataPricingMode` directly here.
Same in other places in this file: you can instantiate `PubdataPricingMode` based on `validiumMode` flag in `scripts/deploy.ts`.

Reply:
Addressed in [eeda60b](https://github.com/matter-labs/era-contracts/pull/225/commits/eeda60b5de12827027e223bf777b87f82eed7672)